extensions/wikia/PageShare/PageShareHooks.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Wikia\PageHeader\Button;
+
+class PageShareHooks {
+	public static function onAfterPageHeaderButtons( &$buttons ) {
+		$title = RequestContext::getMain()->getTitle();
+
+		if ( Wikia::isContentNamespace() && $title->exists() &&
```

Comment:
$title->isContentPage() instead of Wikia::isContentNamespace()? does the same but is less complicated and more object oriented

and probably we should check if title exists before checking namespace. BTW are we sure that $title->exists() check is sufficient? maybe we should use ->isKnown()? see comment here: https://github.com/Wikia/app/blob/3affcfc04852a3c37e0c1885afa58d56d135d3c7/includes/Title.php#L4328-L4339